app/templates/base.html

```diff
@@ -54,6 +54,9 @@
                         <li class="nav-item">
                             <a class="nav-link" href="{{ url_for('search') }}">Search</a>
                         </li>
+                        <li class="nav-item">
+                            <a class="nav-link fw-bolder" href="https://forums.pythonic.guru/">The Forum</a>
+                        </li>
```

Comment:
Although it's nice to have a link to the forum, this app is not intended for us only.